native/comrak_nif/src/lib.rs

```diff
@@ -322,3 +322,11 @@ fn do_safe_html(
     html.replace("&amp;lbrace;", "&lbrace;")
         .replace("&amp;rbrace;", "&rbrace;")
 }
+
+
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn text_to_anchor(env: Env<'_>, text: String) -> NifResult<Term<'_>> {
```

Comment:
Since `fn anchorize` returns `-> String` you could just return a String here:

```suggestion
pub fn text_to_anchor(env: Env<'_>, text: String) -> String {
```